{{cookiecutter.project_slug}}/.github/workflows/ci.yml

```diff
@@ -31,6 +31,9 @@ jobs:
         with:
           python-version: "3.10"
           cache: pip
+          cache-dependency-path: |
+            requirements/base.txt
+            requirements/local.txt
```

Comment:
The `ache-dependency-path` was removed in #4151 because the following actions aren't installing from these requirements, but I didn't realise that it breaks because the default doesn't match anything.

We should just disable caching in this case:

```suggestion
```